index.html

```diff
@@ -5,9 +5,171 @@
+      </nav>
+    </header>
+    <section class="hero">
+      <div class="bg-image">
+        <div class="blur">
+          <div class="main-text">
+            <h1 class="hello-text">Hello! Gamers</h1>
+            <div class="big-text">
+              <h2 class="event">GAMESCOM FALL 2021</h2>
+              <h3 class="date-text">November 15-17</h3>
+            </div>
+            <button type="button" class="buy-tickets">BUY TICKETS</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="program">
+      <div class="happening">
+        <ul class="list">
+          <li data-aos="fade-up" class="items">
+            <img src="images/calendar.png" alt=""><br>
+             <h5>November 17-19</h5>
+          </li>
+          <li data-aos="fade-down" class="items">
+            <img src="images/user.png" alt=""><br>
+           <h5>3500+ &nbsp;Attendees</h5>
+          </li>
+          <li data-aos="fade-up" class="items">
+            <img src="images/globe.png" alt=""><br>
+            <h5>Worldwide</h5>
+          </li>
+          <li data-aos="fade-down" class="items">
+            <img src="images/marketing.png" alt=""><br>
+             <h5>80+ &nbsp;Speakers</h5>
+          </li>
+          <li data-aos="fade-up" class="items">
+            <img src="images/gamepad.png" alt=""><br>
+            <h5>100+ &nbsp;Games</h5>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="speakers">
+        <h2>SPEAKERS</h2>
+        <hr class="red-line">
+        <div id="speakers" class="speaker-card">
+        </div>
+    </section>
+    <section class="partners">
+      <div class="partner-text">
+        <h2>PARTNERS</h2>
+      </div>
+      <hr class="red-line">
+      <div class="partner-card-wrap">
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/squere.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/capcom.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/infinity.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/sega.jpg" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bethesda-2.png" alt="">
+          </div>
+        </div>
+        <div class="partners-container">
+          <div class="partner-cards">
+            <img class="cards-images" src="images/bungie.jpg" alt="">
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="contact" id="contact">
```

Comment:
- [ ] Please make sure that the partners are in only one row and use the same background as the original desgin.

![Capture](https://user-images.githubusercontent.com/64173189/142461831-9c3406a4-b596-4e5f-9a36-dcb86f61efdf.JPG)


Original design

![Capture](https://user-images.githubusercontent.com/64173189/142461736-c90dd3fb-e3b8-46fb-85a1-a2e18d3aaddd.JPG)